source/deploy/mattermost-changelog.md

```diff
@@ -15,6 +15,103 @@ From Mattermost v9.2, this changelog summarizes updates for the latest cloud and
 :depth: 2
 ```
 
+## Release v9.3 - [Feature Release](https://docs.mattermost.com/upgrade/release-definitions.html#feature-release)
+
+**Release Date: December 15, 2023**
+
+### Important Upgrade Notes
+ - Please read the [Important Upgrade Notes](https://docs.mattermost.com/upgrade/important-upgrade-notes.html) before upgrading.
+
+### Compatibility
+ - Updated minimum required Firefox version to v115+.
+ - Updated minimum supported Chromium version to 118+.
+
+### Improvements
+
+#### User Interface (UI)
+ - Updated pre-packaged Playbooks plugin version to [v1.39.1](https://github.com/mattermost/mattermost-plugin-playbooks/releases/tag/v1.39.1).
+ - Updated pre-packaged Calls version to [v0.21.1](https://github.com/mattermost/mattermost-plugin-calls/releases/tag/v0.21.1).
+ - Updated pre-packaged Jira plugin version to [v4.0.1](https://github.com/mattermost/mattermost-plugin-jira/releases/tag/v4.0.1). Also see [v4.0.0](https://github.com/mattermost/mattermost-plugin-jira/releases/tag/v4.0.0) for recent breaking changes.
+ - Added Vietnamese (Beta) as a new language.
+ - Added the ability to passively track keywords with highlights without triggering a notification.
```

Comment:
Remind me, do we usually callout in the changelog on whether the feature is paid or not? @amyblais

Reply:
@johndavidlugtu Yes, what should I add here?

Reply:
A callout that it this feature is available for Professional and Enterprise customers only